pkg/controllers/federatedcluster/clusterjoin.go

```diff
@@ -225,7 +225,7 @@ func handleNotJoinedCluster(
 
 	// 5. Cluster is joined, update condition
 
-	logger.Info("Cluster joined successfully")
+	logger.V(3).Info("Cluster joined successfully")
```

Comment:
```suggestion
	logger.V(2).Info("Cluster joined successfully")
```
This is pretty important information, and does not have the risk of spamming. I think we can log it at 2